recipes/pvlib-python/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: pvlib-python
+  version: {{ version }}
+
+source:
+  fn: pvlib-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/58/3d/8431a36635ad7a06c2fa00c53769074279364bea74ab87f99389b8a90a45/pvlib-{{ version }}.tar.gz
+  md5: 552fcf215911b5bb7b7a2e5ad6a9a6e6
+
+build:
+  number: 0
```

Comment:
You can get rid of build.sh and bld.bad in favor of adding a `script` key to the build section so that it looks like this:

```yaml
build:
  number: 0
  script: python setup.py install --single-version-externally-managed --record=record.txt
```